expression/expression.go

```diff
@@ -1005,9 +1005,12 @@ func scalarExprSupportedByFlash(function *ScalarFunction) bool {
 		ast.TimestampDiff, ast.DateFormat, ast.FromUnixTime,
 		ast.JSONLength:
 		return true
-	case ast.Substr, ast.Substring:
+	case ast.Substr, ast.Substring, ast.Left, ast.Right, ast.CharLength:
 		switch function.Function.PbCode() {
 		case
+			tipb.ScalarFuncSig_LeftUTF8,
+			tipb.ScalarFuncSig_RightUTF8,
+			tipb.ScalarFuncSig_CharLengthUTF8,
```

Comment:
Is this `char length` related to what we are doing?

Reply:
open it if needed work around or check when bugs occur